### DISCUSSION TITLE
Prebuilt onnxruntime containers

### DISCUSSION THREAD

**Discussion by ramkrishna1121** [microsoft/onnxruntime#14171] [Other Q&A]
Are prebuilt onnxruntime containers available that be used for performance benchmarking of onnx models? Or do they have to be built from source?

**Comment by snnn** [comment]
No, there isn't any.  But you don't need to build everything from source. In your docker file you can install prebuilt packages from pypi.org.

**Comment by jack-gits** [comment]
have any instruction or manual?

**Comment by ramkrishna2910** [comment]
To install pre-built pypi packages as mentioned by @snnn add `RUN pip install <package>` to your dockerfile.
Instructions to build onnxruntime containers from src, refer to this [link](https://github.com/microsoft/onnxruntime/blob/main/dockerfiles/README.md)
